### PR TITLE
✨ Add RELEASE.md

### PR DESCRIPTION
**Pull request by laurentsimon**
Add RELEASE.md for Golan builder.

### CODE REVIEW

**Comment by laurentsimon** [comment]
@asraa PTAL. Don't freak out ... there are a few manual things to do. We need to update the part on verifier's e2e tests.
I think we can automate some of these.

Let's do the v1 release together, so we can walk thru the steps :-)

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+Send a pull request to merge the changes into the verifier's repository. The pre-submits will validate that the verifier is able to verify provenance from the `$BUILDER_TAG` builder.
+
+## Finalize release
+
+Untick the `This is a pre-release` option. That's it!
+
+## Update the starter workflow
+
+TODO
```
Comment (ianlewis) [inline RELEASE.md:134]: You can probably comment these out as todos and put in an issue to update them.

**Review by ianlewis — APPROVED** [review]
LGTM

File: RELEASE.md (hunk 1 above)
Comment (laurentsimon) [reply RELEASE.md]: Created the issues.

**Pull request merged**